docs/housekeeping-failed-webhooks.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Failed Webhooks
+permalink: /housekeeping-failed-webhooks
+---
+
+<div class="chart-placeholder">
+	<h3>Failed Webhooks</h3>
+	<canvas
+		data-url="{{ site.dataURL }}/failed-webhooks.tsv"
+		data-type="history"
+		data-config='{
+			"views":
+			[
+				{
+					"label": "2 m",
+					"tooltip": "Show the last 2 months",
+					"aggregate": false,
+					"slice": [0, 61],
+					"default": true
+				},
+				{
+					"label": "2 y",
+					"tooltip": "Show the last 2 years",
+					"aggregate":
+					{
+						"period": "week",
+						"method": "sum"
+					},
+					"slice": [0, 106]
+				},
+				{
+					"label": "all",
+					"tooltip": "Show all data",
+					"aggregate":
+					{
+						"period": "week",
+						"method": "sum"
+					}
+				}
+			]
+    }'></canvas>
+	<div class="info-box"></div>
+</div>
+
+<div class="chart-placeholder">
+  <table data-url="{{ site.dataURL }}/failed-webhooks-detailed.tsv" data-type="table"></table>
+
+	<div class="info-box">
+		<p>
+			<ul>
```

Comment:
We might want to add some text here explaining why webhooks could fail (e.g. because the server is down, DNS resolution is not possible, webhook server error, ...) ... but we can work on this in a subsequent PR I think

Reply:
Isn't that already resolved by the `message` column in the table, @larsxschneider?

Reply:
Very briefly, yes. We usually try to give admins some additional context in that column. E.g. a useful thing would be to teach team about `ghe-webhook-logs` to investigate a webhook error furthe. In the table then will see the failing hook id and then they could run to learn more:
```
ghe-webhook-logs --failures --hook-id <hook-id> 
```

Reply:
Will open a new PR for this soon™